packages/core/src/components/FormatLargeNumber/FormatLargeNumber.tsx

```diff
@@ -26,8 +27,8 @@ export default function FormatLargeNumber(props: Props) {
   const formatedValue = useMemo(() => {
     if (typeof value === 'undefined' || value === null) {
       return value;
-    } else if (value instanceof Big) {
-      return value.toNumber().toLocaleString(locale);
+    } else if (value instanceof BigNumber) {
```

Comment:
That is not true because you are using packages in the wrong way. You have two separate node_modules one in packageA and the second version is in packageB. This is the wrong way how to work with packages. The same problem you can have with React as well. Everything related to this is fixed with Lerna monorepo and one package.json